lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -1205,6 +1216,11 @@ private Task claim(String taskId, boolean forceClaim)
       throws TaskNotFoundException, InvalidStateException, InvalidOwnerException,
           NotAuthorizedException {
     String userId = taskanaEngine.getEngine().getCurrentUserContext().getUserid();
+    User user = userMapper.findById(userId);
```

Comment:
Here we should also check if the additonal user data is activated or not, like in the completeTasks method above

Reply:
Ok :) I added it